samples/test-dal-with-nunit/DalWithNunitSample/DAL/IPizzaService.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System.Collections.Generic;
+using System.Threading.Tasks;
+
+namespace DAL
+{
+    public interface IPizzaService
```

Comment:
This is a repository, not a service.

Reply:
Does it make sense to introduce repository over the `DbContext` ?

Reply:
Nothing in this example says Service to me.  It's pure CRUD stuff you would see in a generic `IRepository<T>`.  On the upside, it's not leaking `IQueryable<T>` and its using Task API.

Reply:
BLL would be something like `Task OrderPizzaPie(List<Topping> toppings, string specialInstructions);`

Reply:
Let's just change methods of this service to be more 'business specific' and don't introduce any repositories over DbContext. The purpose of this merge request is to provide sample of usage of this library with 'just class library' integration tests (for example business logic assembly from wpf or asp.net mvc 5, without actual introducing these specific projects).

Reply:
I guess I am passionate about following clean, modular practices.  DbContext tightly couples you to Entity Framework.  As such, it's not going to test the DAL in a way that is both easy to integration test and unit test.  That should be the whole point of the example.  I face far too many developers who read stupid blog posts saying things like "Repository pattern is dead" who have never had to clean up their own mess.  Open source projects like these should be the gold standard for others to follow.

Reply:
In order to avoid discussions about ideal repository i refactor it to be simple service

Reply:
That doesn't solve the problem.  Then your BLL is directly accessing your database access layer.  The whole point of the repository pattern is to decouple BLL from DAL so that you can:
1. Write pure unit tests on your repository
2. Write integration tests on your repository